### ISSUE TITLE
doccumentation suggests inline comments for configuration, but that doesn't seem to work

### ISSUE THREAD

**Issue by evgeni** [ansible/ansible-lint#2046]
##### Summary

When you run `ansible-lint` and get an error, you also get helpful output how to ignore said error:

```
        You can skip specific rules or tags by adding them to your configuration file:
        # .config/ansible-lint.yml
        warn_list:  # or 'skip_list' to silence them completely
          - fqcn-builtins  # Use FQCN for builtin actions.
```

However, adding this exactly like that to the config doesn't work.

```yaml
warn_list:
    - fqcn-builtins  # Use FQCN for builtin actions.
```

Removing the comment fixes the issue:

```yaml
warn_list:
    - fqcn-builtins
```

The same is true for the docs at https://ansible-lint.readthedocs.io/en/latest/configuring/ -- they contain inline comments, which `ansible-lint` doesn't seem to like.

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
ansible [core 2.12.3]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/evgeni/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/evgeni/Devel/theforeman/foreman-ansible-modules/venv/lib64/python3.10/site-packages/ansible
  ansible collection location = /home/evgeni/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/evgeni/Devel/theforeman/foreman-ansible-modules/venv/bin/ansible
  python version = 3.10.2 (main, Jan 17 2022, 00:00:00) [GCC 11.2.1 20211203 (Red Hat 11.2.1-7)]
  jinja version = 3.0.3
  libyaml = False

% ansible-lint --version
ansible-lint 6.0.2 using ansible 2.12.3
```

- ansible installation method: pip
- ansible-lint installation method: pip

##### OS / ENVIRONMENT

Fedora, but should be irrelevant

##### STEPS TO REPRODUCE

See above.

##### Desired Behavior

Users can copy paste from docs.

##### Actual Behavior

Users get invalid configurations
